values.yaml

```diff
@@ -895,7 +895,33 @@ jobs:
   # For example, if an upgrade only touches RBAC config, use this to disable other jobs:
   #   helm upgrade ... --set 'jobs.skip={apikey_load,key_load,register_content}'
   skip: []
-
+  #
+  # Advanced: Add extra Helm hook Jobs
+  # These hook jobs will use the same settings (eg image, annotations, pod placement) as the other jobs.
+  # They will have st2 cli configured, st2.conf files, and packs volumes mounted.
+  # See available hooks list: https://helm.sh/docs/topics/charts_hooks/#the-available-hooks
```

Comment:
This is powerful, but I think the use case examples would be nice to include somewhere. Otherwise, very few would know about it and use it.
Perhaps a Readme section describing this feature with some examples?

Reply:
That makes sense. I will add a README section.